src/solver.jl

```diff
@@ -5,6 +5,7 @@
 #===================================================
  This file implements the default PajaritoSolver
 ===================================================#
+@enum disagg_soc _disagg_soc_default _disagg_soc_on _disagg_soc_off
```

Comment:
``disagg_soc`` is the name of the type, you can use it like ``disaggregate_soc::disagg_soc``. Also since it's a type it should start with upper case.